packages/common/src/models/dictionary_categories.ts

```diff
@@ -2,20 +2,28 @@ import { integer, pgTable, serial, timestamp, varchar } from 'drizzle-orm/pg-cor
 
 import { relations } from 'drizzle-orm/relations';
 import { dictionaries } from './dictionaries.js';
+import { submissions } from './submissions.js';
+import { submittedData } from './submitted_data.js';
 
 export const dictionaryCategories = pgTable('dictionary_categories', {
 	id: serial('id').primaryKey(),
+	activeDictionaryId: integer('active_dictionary_id'),
```

Comment:
not sure what the difference is for these two properties.

is the "activeDictionaryId" implying the category has multiple dictionaries that belong to it, but there's going to be a single dictionary in this category that is going to be active?
does that mean other categories will have their own "active" dictionary?

Reply:
Discussed over zoom with @leoraba and @joneubank. 
My understanding in the last comment is accurate, and it's just the naming choice of  "dictionary" vs "dictionary category" that may be semantically confusing, and we need to explain through documentation

Reply:
As discussed and just to clarify here, it is right to say that each category is going to have only one active dictionary. Is reasonable to add proper documentation about these terminology.